.github/workflows/ci.yml

```
@@ -116,6 +116,11 @@ jobs:
           name: coverage
           path: ./tests/merged-coverage/coverage.json
           if-no-files-found: error
+      - uses: actions/upload-artifact@v3
+        with:
+          name: coverage-lcov
+          path: ./tests/merged-coverage/lcov.info
+          if-no-files-found: error
```

Nit: Perhaps it would be better to upload all reports at once?

```suggestion
          path: ./tests/merged-coverage
          if-no-files-found: error
```